### ISSUE TITLE
bug: Unable to bring up the keyboard when modifying the package name with TalkBack

### ISSUE THREAD

**Issue by bzkmsjy** [MorpheApp/morphe-manager#995]
### Bug description

Hello, I am using this software with TalkBack. When patching in Expert Mode, I wanted to modify the settings of the Clone app patch. When editing the Package name field, TalkBack cannot properly bring up the on-screen keyboard.

When I double-tap the Default button, it only opens a drop-down menu containing another button also called Default. I can only select it, which returns me to the original state, and no keyboard appears for me to edit the package name.

Later, I paused TalkBack and tried directly touching the option area. The keyboard suddenly appeared and I was able to enter text normally.

This seems to be an accessibility issue. I hope it can be fixed. Thank you!

### Version of Morphe (Manager) and the app you are patching

v1.31.1

### Morphe logs

```shell
No need
```

### Acknowledgements

- [x] I have checked all open and closed bug reports and this is not a duplicate.
- [x] I have chosen an appropriate title.
- [x] All requested information has been provided properly.
- [x] The bug only affects using Morphe (Manager)

**Comment by bzkmsjy** [comment]
I would also like to add another issue with this input field. The field contains the text “default” by default. If I select all of it and delete it, the text “default” appears again in the input field.

Therefore, the user has to enter the desired package name first and then delete “default” in order to get the desired result.

**Comment by MarcaDian** [comment]
It should be fixed in the next dev release.
